make/test.mk

```diff
@@ -157,19 +157,30 @@ get-and-publish-host-operator:
 ###########################################################
 
 .PHONY: deploy-members
-deploy-members: create-member1 create-member2 get-and-publish-member-operator
+deploy-members: create-member1 create-member2 get-and-publish-member-operator create-member-resources
 
 .PHONY: create-member1
 create-member1:
 	@echo "Deploying member operator to $(MEMBER_NS)..."
 	$(MAKE) create-project PROJECT_NAME=${MEMBER_NS}
+	-oc label ns ${MEMBER_NS} app=member-operator
 
 .PHONY: create-member2
 create-member2:
 ifeq ($(SECOND_MEMBER_MODE),true)
 	@echo "Deploying second member operator to ${MEMBER_NS_2}..."
 	$(MAKE) create-project PROJECT_NAME=${MEMBER_NS_2}
-	oc apply -f deploy/member2-operator/member-operator-config-map.yaml -n ${MEMBER_NS_2}
+	-oc label ns ${MEMBER_NS_2} app=member-operator
+endif
+
+.PHONY: create-member-resources
+create-member-resources:
+ifeq ($(MEMBER_REPO_PATH),)
+	$(eval MEMBER_REPO_PATH = /tmp/codeready-toolchain/member-operator)
+endif
+ifeq ($(SECOND_MEMBER_MODE),true)
+	oc apply -f ${MEMBER_REPO_PATH}/config/crd/bases/toolchain.dev.openshift.com_memberoperatorconfigs.yaml
```

Comment:
Right, I was initially calling it before the member operator installation but I found out the hard way that this fails when using the `test-e2e` make target (instead of the `test-e2e-local` because the member operator repo wasn't cloned yet so I moved it to happen post-installation. Ideally I wanted to do it before the install in case the installation completes before we apply the configuration but I didn't see a clean way to do that and I don't think we wait for the installation to complete so the configuration should be in place before the operator actually starts anyway. So yeah, I'll remove it now.

Reply:
Done in e212207845a9e65aa905721c1bc87f4a0f0036dd

Reply:
I want to make the wait script smarter so it would wait for the availability of the CRDs instead of serviceaccounts - this would make this part much safer.